### PR TITLE
Code and comment clarifications, add csv format

### PR DESCRIPTION
**Pull request by bpbond**

### CODE REVIEW

File: R_scripts/licordat.R [hunk 1]
```diff
@@ -78,38 +78,48 @@ read_dir <- function(path) {
...
+         Dest_Elevation <- factor(Dest_Elevation, levels = HML),
+         Date = paste(month(Timestamp), "/", day(Timestamp)),
+         Group = paste(Origin_Plot, "->", Destination_Plot),
+         Group = if_else(Experiment == "Control", "Control", Group)) ->
+  licorDat
 
-save(licorDat, file = "../outputs/licordat.rda")
\ No newline at end of file
+cat("Saving data...\n")
+write_csv(licorDat, "../outputs/licor_data.csv")
```
Comment (bpbond) [inline R_scripts/licordat.R]: @stephpenn1 - in general we want to save using human-readable formats like CSV, not RDA

Comment (stephpenn1) [reply R_scripts/licordat.R]: This was saved as an RDA to preserve the column formats. If you remember the issue I had ordering the plots using facet_grid.. we decided to save as RDA instead. I can save it as RDA and CSV?

Comment (bpbond) [reply R_scripts/licordat.R]: Hmm. Ah, OK, thanks. Let's save as both then; it's really useful to have a human-readable format.

**Comment by bpbond** [comment]
Look good @stephpenn1 ?

**Comment by stephpenn1** [comment]
Looks great

**Comment by stephpenn1** [comment]
Can I try accepting the pull request? Do i click “Merge pull request” if so?

**Comment by bpbond** [comment]
Crap, sorry, my bad. Of course. Next time you approval and merge.

**Pull request merged**